README.md

```diff
@@ -412,6 +412,7 @@ Throughout this list you'll see next to each resource and emoji. Here's what eac
 - :video_camera: [DSA + C++ Interview Preparation Course](https://www.youtube.com/watch?v=z9bZufPHFLU&list=PLfqMhTWNBTe0b2nM6JHVCnAkhQRGiZMSJ)
 - :video_camera: [Data Structures](https://www.youtube.com/playlist?list=PL2_aWCzGMAwI3W_JlcBbtYTwiQSsOTa6P)
 - :green_book: [Minimum Stack / Minimum Queue](https://cp-algorithms.com/data_structures/stack_queue_modification.html)
+- :page_facing_up: [Data Structures Tutorial | Scalar](https://www.scaler.com/topics/data-structures/)
```

Comment:
According to the [Contribution Guidelines](https://github.com/shahednasser/awesome-resources/blob/master/CONTRIBUTING.md) you're supposed to use the 📚 emoji.

Reply:
hi!
I've updated the emoji :)